Editor/Editors/CinemachineSplineDollyEditor.cs

```diff
@@ -0,0 +1,96 @@
+#if CINEMACHINE_UNITY_SPLINES
+using System;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Splines;
+
+namespace Cinemachine.Editor
+{
+    [CustomEditor(typeof(CinemachineSplineDolly))]
+    [CanEditMultipleObjects]
+    sealed class CinemachineSplineDollyEditor : BaseEditor<CinemachineSplineDolly>
+    {
+        public override void OnInspectorGUI()
+        {
+            BeginInspector();
+            bool needWarning = false;
+            for (int i = 0; !needWarning && i < targets.Length; ++i)
+                needWarning = (targets[i] as CinemachineSplineDolly).m_Spline == null;
+            if (needWarning)
+                EditorGUILayout.HelpBox("A Path is required", MessageType.Warning);
+
+            needWarning = false;
+            for (int i = 0; !needWarning && i < targets.Length; ++i)
+                needWarning = (targets[i] as CinemachineSplineDolly).m_AutoDolly.m_Enabled 
+                    && (targets[i] as CinemachineSplineDolly).FollowTarget == null;
+            if (needWarning)
+                EditorGUILayout.HelpBox("AutoDolly requires a Follow Target", MessageType.Warning);
+
+            DrawRemainingPropertiesInInspector();
+        }
+
+        [DrawGizmo(GizmoType.Active | GizmoType.NotInSelectionHierarchy
+                | GizmoType.InSelectionHierarchy | GizmoType.Pickable, typeof(CinemachineSplineDolly))]
+        static void DrawGizmos(CinemachineSplineDolly dolly, GizmoType selectionType)
+        {
+            if (Selection.activeGameObject == dolly.VirtualCamera.gameObject)
+                DrawDollyGizmo(dolly, Color.green, 1); // GML todo: make this configurable
```

Comment:
Do you have any thoughts on how we should deal with this hardcoding?

Reply:
We could add a color to Cinemachine preferences or use Boundary Object color.

Reply:
![Screen Shot 2022-02-16 at 2 06 16 PM](https://user-images.githubusercontent.com/57672095/154337627-7f9d5e49-1154-47df-b104-08aaaf65a72f.png)